packages/flutter/test/widgets/semantics_test.dart

```diff
@@ -524,7 +524,8 @@ void main() {
       ..remove(SemanticsAction.moveCursorForwardByWord)
       ..remove(SemanticsAction.moveCursorBackwardByWord)
       ..remove(SemanticsAction.customAction) // customAction is not user-exposed.
-      ..remove(SemanticsAction.showOnScreen); // showOnScreen is not user-exposed
+      ..remove(SemanticsAction.showOnScreen) // showOnScreen is not user-exposed
+      ..removeWhere((SemanticsAction action) => action.index == 1 << 23);
```

Comment:
Can you add a todo so that we don't forget this?

Reply:
I'll only merge this if the other PR is ready so I can immediately update the test, sounds good?

Reply:
ok, sgtm

Reply:
@chunhtai do you think can merge both PRs now?

Reply:
let me take a quick look at the engine pr now

Reply:
lgtm'd